### PR TITLE
Add pre-built binaries for bz2, gdbm and openssl

### LINKED ISSUE #3123: Error building openssl
## Description
Tried doing `crew upgrade` after a while and I'm now seeing an error building openssl; something about not being able to find libc.

If you executed a specific command:
```shell
crew upgrade
```

Output:
```shell
/usr/local/bin/ld: libcrypto.a(dso_dlfcn.o): in function `dlfcn_globallookup':
dso_dlfcn.c:(.text+0x11): warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/bin/ld: cannot find /usr/local/lib64/libc.so.6
/usr/local/bin/ld: cannot find /usr/local/lib64/ld-linux-x86-64.so.2
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile.shared:169: link_a.gnu] Error 1
make[4]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p'
make[3]: *** [Makefile:357: do_linux-shared] Error 2
make[3]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p'
make[2]: *** [Makefile:310: libcrypto.so.1.0.0] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p'
make[1]: *** [Makefile:109: shared] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2p.tar.gz.dir/openssl-OpenSSL_1_0_2p/crypto'
make: *** [Makefile:287: build_crypto] Error 1
openssl failed to install: `make V=0 -j4` exited with 2
```

## Versions and system information
chromebrew: `crew --version`
```
chronos@localhost ~/julia $ crew --version
1.1.2
```

ChromeOS: `cat /etc/lsb-release`
```
chronos@localhost ~/julia $ cat /etc/lsb-release
CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=5151026
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=0
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R72-11307.0.0
CHROMEOS_RELEASE_BUILD_NUMBER=11307
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=72
CHROMEOS_RELEASE_DESCRIPTION=11307.0.0 (Official Build) dev-channel eve 
CHROMEOS_RELEASE_KEYSET=mp
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=dev-channel
CHROMEOS_RELEASE_VERSION=11307.0.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=11307.0.0
```
Model: Pixelbook

Architecture: `uname -m`
```
chronos@localhost ~/julia $ uname -m
x86_64
```

### PR DESCRIPTION
**Pull request by uberhacker**
Fixes #3123.

Tested on:
- [x] aarch64
- [x] armv7l
- [x] i686
- [x] x86_64

### CODE REVIEW

File: packages/mandb.rb [hunk 1]
```diff
@@ -54,10 +54,6 @@ def self.install
   end
 
   def self.postinstall
-    puts
-    puts "To finish the installation, set the default MANPATH environment variable:".lightblue
-    puts "echo \"export MANPATH=#{CREW_PREFIX}/man:$MANPATH\" >> ~/.bashrc".lightblue
```
Comment (JL2210) [inline packages/mandb.rb]: Yay! This is actually what I was trying to remove with #3002.

**Comment by JL2210** [comment]
You mentioned that there wouldn't be any serious repercussions if you merged your `update-glibc-2.23-2.27` branch. Maybe we should do that just to stop these erroneous issues?

**Comment by uberhacker** [comment]
I don't want to merge that branch until release 72 becomes stable.  The reason being we would be telling a lot of folks to reinstall it afterwards.

File: packages/mandb.rb (hunk 1 above)
Comment (uberhacker) [reply packages/mandb.rb]: Thanks to @superloach for following through on this.

**Comment by JL2210** [comment]
Not really. We should be able to just upgrade, if it has binaries. `glibc` is backwards compatible, so there wouldn't be any issue with that.

**Comment by JL2210** [comment]
You know, with musl, we wouldn't be having these issues. If anything, we'd probably gain some ease of upgrade with their advertised "atomic updates".

Notes from Wikipedia:
> An atomic transaction is an indivisible and irreducible series of database operations such that either all occur, or nothing occurs.
> an operation appears to occur at a single instant between its invocation and its response

**Pull request merged**